From: Michael Tokarev <mjt@tls.msk.ru>
Subject: use libresolv on non-linux too
Date: Fri, 04 Nov 2022 18:10:18 +0300

nslookup uses symbols from libresolv, no matter it is
on linux or not. This also eliminates hackish way to
detect linux.

diff --git a/Makefile.flags b/Makefile.flags
index c34356230..f26ce719e 100644
--- a/Makefile.flags
+++ b/Makefile.flags
@@ -186,6 +186,4 @@ endif
 ifeq ($(CONFIG_FEATURE_NSLOOKUP_BIG),y)
-ifneq (,$(findstring linux,$(shell $(CC) $(CFLAGS) -dumpmachine)))
 LDLIBS += resolv
-endif
 endif
 
